tests/Fixer/Whitespace/MethodChainingIndentationFixerTest.php

```diff
@@ -186,6 +186,24 @@ public function provideFixCases()
 ->bar()
 ;',
             ],
+            [
+                '<?php
+
+    $user->setEmail(\'voff.web@gmail.com\')
```

Comment:
use `"` instead of `\'` for readability ;)

Reply:
I just copied from other test cases, btw done.